tests/ops/test_qubit_ops.py

```diff
@@ -777,6 +778,241 @@ def test_isingxx_decomposition(self, tol):
 
+    device_methods = [
+        ["default.qubit", "finite-diff"],
+        ["default.qubit", "parameter-shift"],
+        ["default.qubit", "backprop"],
+        ["default.qubit", "adjoint"],
+    ]
+
+    phis = [0.1, 0.2, 0.3]
+
+    configuration = []
+
+    for phi in phis:
+        for device, method in device_methods:
+            configuration.append([device, method, phi])
+
+    @pytest.mark.parametrize("dev_name,diff_method,phi", configuration)
+    def test_isingxx_autograd_grad(self, tol, dev_name, diff_method, phi):
+        """Test the gradient for the gate IsingXX."""
+        dev = qml.device(dev_name, wires=2)
+
+        psi_0 = 0.1
+        psi_1 = 0.2
+        psi_2 = 0.3
+        psi_3 = 0.4
+
+        init_state = npp.array([psi_0, psi_1, psi_2, psi_3], requires_grad=False)
+        norm = np.linalg.norm(init_state)
+        init_state /= norm
+
+        @qml.qnode(dev, diff_method=diff_method, interface="autograd")
+        def circuit(phi):
+            qml.QubitStateVector(init_state, wires=[0, 1])
+            qml.IsingXX(phi, wires=[0, 1])
+            return qml.expval(qml.PauliZ(0))
+
+        phi = npp.array(0.1, requires_grad=True)
+
+        expected = (
+            0.5
+            * (1 / norm ** 2)
+            * (
+                -np.sin(phi) * (psi_0 ** 2 + psi_1 ** 2 - psi_2 ** 2 - psi_3 ** 2)
+                + 2
+                * np.sin(phi / 2)
+                * np.cos(phi / 2)
+                * (-(psi_0 ** 2) - psi_1 ** 2 + psi_2 ** 2 + psi_3 ** 2)
+            )
+        )
+
+        res = qml.grad(circuit)(phi)
+        assert np.allclose(res, expected, atol=tol, rtol=0)
+
+    @pytest.mark.parametrize("dev_name,diff_method,phi", configuration)
+    def test_isingzz_autograd_grad(self, tol, dev_name, diff_method, phi):
+        """Test the gradient for the gate IsingZZ."""
+        dev = qml.device(dev_name, wires=2)
+
+        psi_0 = 0.1
+        psi_1 = 0.2
+        psi_2 = 0.3
+        psi_3 = 0.4
+
+        init_state = npp.array([psi_0, psi_1, psi_2, psi_3], requires_grad=False)
+        norm = np.linalg.norm(init_state)
+        init_state /= norm
+
+        @qml.qnode(dev, diff_method=diff_method, interface="autograd")
+        def circuit(phi):
+            qml.QubitStateVector(init_state, wires=[0, 1])
+            qml.IsingZZ(phi, wires=[0, 1])
+            return qml.expval(qml.PauliX(0))
+
+        phi = npp.array(0.1, requires_grad=True)
+
+        expected = (1 / norm ** 2) * (-2 * (psi_0 * psi_2 + psi_1 * psi_3) * np.sin(phi))
+
+        res = qml.grad(circuit)(phi)
+        assert np.allclose(res, expected, atol=tol, rtol=0)
+
+    @pytest.mark.parametrize("dev_name,diff_method,phi", configuration)
+    def test_isingxx_jax_(self, tol, dev_name, diff_method, phi):
+        """Test the gradient for the gate IsingXX."""
+
+        if diff_method in {"finite-diff"}:
+            pytest.skip("Test does not support parameter-shift")
```

Comment:
The message seems to be off here. Would this be the same issue as for parameter-shift when using `jax`?
```suggestion
            pytest.skip("Test does not support finite-diff")
```